iotdb-client/service-rpc/pom.xml

```diff
@@ -121,7 +121,7 @@
         <dependency>
             <groupId>org.apache.iotdb</groupId>
             <artifactId>iotdb-thrift</artifactId>
-            <version>${project.version}</version>
+            <version>1.3.0-SNAPSHOT</version>
```

Comment:
Same point as above.

Reply:
I just stumbled of the the usage in those two cases, but would lean towards doing that for all.

Reply:
Undid the change ...